tasks/release.py

```diff
@@ -1248,13 +1250,20 @@ def create_rc(ctx, major_versions="6,7", patch_version=False, upstream="origin")
             code=1,
         )
 
-    create_pr(
+    pr_url = create_pr(
         f"[release] Update release.json and Go modules for {versions_string}",
         current_branch,
         update_branch,
         new_final_version,
     )
 
+    # Step 4 - If slack workflow webhook is provided, send a slack message
+    if slack_webhook:
+        print(color_message("Sending slack notification", "bold"))
+        ctx.run(
+            f"curl -X POST -H 'Content-type: application/json' --data '{{\"pr_url\":\"{pr_url}\"}}' {slack_webhook}"
```

Comment:
```suggestion
            f"curl -X POST -H 'Content-Type: application/json' --data '{{\"pr_url\":\"{pr_url}\"}}' {slack_webhook}"
```

Reply:
In addition, does `'{{\"pr_url\":\"{pr_url}\"}}'` work properly in the fstring ?

Reply:
I checked this curl with fake invoke task and it works ok. 
As far as I know the headers are case insensitive so I write it this way by default but I guess it could be changed for the consistency reason.